opentimelineio_contrib/adapters/extern_rv.py

```diff
@@ -207,8 +211,21 @@ def _write_timeline(tl, to_session):
 def _create_media_reference(mr, to_session):
     if hasattr(mr, "media_reference") and mr.media_reference:
         if isinstance(mr.media_reference, otio.schema.ExternalReference):
-            to_session.setMedia([str(mr.media_reference.target_url)])
+            media = [str(mr.media_reference.target_url)]
+            
+            if mr.parent() and mr.parent().kind == otio.schema.TrackKind.Audio:
+                # Create blank video media to accompany audio for valid source
```

Comment:
can you also add a note here for the fact that putting it *after* the audio means that it puts the correct name onto the RVFileSource?

Reply:
Sure

Reply:
Added comment in latest commit